frontend/packages/gitops-plugin/src/components/list/GitOpsTableRow.tsx

```diff
@@ -41,12 +119,65 @@ const GitOpsTableRow: RowFunction<GitOpsAppGroupData> = (props) => {
           <span style={{ marginRight: 'var(--pf-global--spacer--xs)' }}>{appGroup.repo_url}</span>
         </ExternalLink>
       </TableData>
-      <TableData className={tableColumnClasses[2]}>{appGroup.environments.join(', ')}</TableData>
+      <TableData className={classNames(tableColumnClasses[2], 'pf-u-text-nowrap')}>
+        {appGroup && appGroup.sync_status ? (
+          <Flex className="syncStatus">
+            <GetSyncFragment tooltip={ttSynced} count={numOfSynced} icon="check" />
+            <GetSyncFragment tooltip={ttOutOfSync} count={numOfOutOfSync} icon="exclamation" />
+            <GetSyncFragment tooltip={ttUnknown} count={numOfUnknown} icon="unknown" />
+          </Flex>
+        ) : (
+          <span>{appGroup.environments.join(', ')}</span>
+        )}
+      </TableData>
       <TableData className={tableColumnClasses[3]}>
-        {/* this is just a placeholder until backend changes can go in to get this data */}-
+        {latestDeployedTime ? (
+          <Flex>
+            <FlexItem className="lastDeploymentTime" spacer={{ default: 'spacerXs' }}>
+              <span>
+                <Timestamp timestamp={latestDeployedTime} />
+              </span>
+            </FlexItem>
+            <FlexItem>{latestDeployedEnv}</FlexItem>
+          </Flex>
+        ) : (
+          <span>-</span>
+        )}
       </TableData>
     </TableRow>
   );
 };
 
+const GetSyncFragment: React.FC<SyncProps> = ({ tooltip, count, icon }) => {
```

Comment:
One Component per file please. Move this to a different file and import it.

Reply:
Done